src/ispd/gui/UserConfigurationDialog.java

```diff
@@ -149,18 +149,22 @@ private void onAddClick(final ActionEvent evt) {
         );
 
         // TODO: Parse result and error.
-        final Double result =
-                Double.parseDouble(JOptionPane.showInputDialog(this,
-                "Enter user power comsumption limit"));
+        String sResult ="";
```

Comment:
Merge declaration and initialization in one line; make variable 'final' if applicable.